examples/TID_model_custom_property/Modelling_TID_TNID_example_notebook.ipynb

```diff
@@ -0,0 +1,497 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        print("Time:", sat.local_time, "Dose [rad]: ", dose)
</pre><p>Can you change to</p><p> </p><pre class="ql-syntax" spellcheck="false"> "TID [rad]: ", dose
</pre><p> </p><p>(it is a bit confusing that you use them interchangeably since dose can also be the dose gotten during one step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/168/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>